corehq/apps/callcenter/views.py

```diff
@@ -40,6 +41,7 @@ def static_options(self):
         return []
 
 
+@location_safe
```

Comment:
Ah I see. This doesn't filter users, so sounds like its not location safe. I just added those lines in response to [this comment](https://github.com/dimagi/commcare-hq/pull/13678#discussion_r85122748), but sounds like its unnecessary, so I'll remove it.